build.gradle.kts

```diff
@@ -43,6 +43,13 @@ kotlin {
             }
         }
     }
+    ios {
+        binaries {
+            framework {
+                baseName = "MultiplatformExpressionsEvaluator"
```

Comment:
the name could be just `ExpressionsEvaluator`

Reply:
Updated as suggested :)